### PR TITLE
[WIP] The Docening

### PR DESCRIPTION
**Pull request by sudo-carson**
Massive documentation overhaul

### CODE REVIEW

**Comment by XFaonAE** [comment]
Dont ask my how painfull it was to rev xD
Ty tho :D

File: docs/src/css/custom.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+html[data-theme='light'] .themedImage--light_node_modules-\@docusaurus-theme-classic-lib-next-theme-ThemedImage-styles-module {
+	filter: invert(1);
+}
+
+html[data-theme='dark'] .themedImage--dark_node_modules-\@docusaurus-theme-classic-lib-next-theme-ThemedImage-styles-module {
+	filter: none;
+}
+
+html[data-theme='light'] .navbar__brand:hover .themedImage--light_node_modules-\@docusaurus-theme-classic-lib-next-theme-ThemedImage-styles-module {
+	filter: brightness(0) saturate(100%) invert(64%) sepia(20%) saturate(340%) hue-rotate(154deg) brightness(95%) contrast(90%);
```
Comment (sudo-carson) [inline docs/src/css/custom.css:55]: dont want to talk about this one

Comment (XFaonAE) [reply docs/src/css/custom.css:55]: xD

**Comment by sudo-carson** [comment]
If someone could help me work on the homepage that would be great 🙂

**Comment by sudo-carson** [comment]
Closing in favor of the rewrite's documentation.

**Pull request closed without merging**